tonic-reflection/build.rs

```diff
@@ -0,0 +1,15 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    const OUT_DIR: &str = "src/proto";
```

Comment:
might want to add that `println` that allows us to only call build.rs if the file has changed.